src/sheet_metal_conus.py

```diff
@@ -55,7 +63,18 @@ def __init__(self):
         self.OptionParser.add_option('-d', '--verbose', action = 'store',
             type = 'inkbool', dest = 'verbose', default = False,
             help = 'Enable verbose output of calculated parameters. Used for debugging or is someone needs the calculated values.')
-
+        # Arrowed lines
+        self.dimline_style = {
+            'stroke'        : '#000000',
+            'stroke-width'  : '0.75px',
+            'fill'          : 'none',
+            'marker-start'  : 'url(#ArrowDIN-start)',
+            'marker-end'    : 'url(#ArrowDIN-end)'
+            }
+        # self.dimline_attribs = {'style'  : simplestyle.formatStyle(self.dimline_style),
+                                # inkex.addNS('label','inkscape') : 'dimline',
+                                # 'd' : 'm 0,0 200,0'
+                               # }
```

Comment:
global def of the arrow linestyle. we refer to and change this for solid and dashed lines